### PR TITLE
create: add simple application template

### PR DESCRIPTION
**Pull request by 0x501D**
@TarantoolBot document
Title: `tt create`: a new template single_instance

The template creates a single instance application for Tarantool 3.

### CODE REVIEW

File: cli/cmd/create.go [hunk 1]
```diff
@@ -54,6 +54,10 @@ Built-in templates:
 
     $ tt create <template name> --name app1
 
+# Create minimal Tarantool 3 application.
+
+    $ tt create simple --name new_app
```
Comment (oleg-jukovec) [inline cli/cmd/create.go:59]: In my taste:
```suggestion
# Create a Tarantool 3 application with a single instance configuration.

    $ tt create single_instance --name new_app
```

Comment (0x501D) [reply cli/cmd/create.go:59]: Sounds good.

Comment (0x501D) [reply cli/cmd/create.go]: Renamed

**Review by psergee — APPROVED** [review]
Thank you for the patch!

**Review by oleg-jukovec — COMMENTED** [review]
Please, add the template to the list:

https://github.com/tarantool/tt/blob/ef10db73be1570a3f2895f14e69b4e3f8942dfb1/cli/cmd/create.go#L49-L51

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- `tt create`: added single instance application template
```
Comment (psergee) [inline CHANGELOG.md:12]:
```suggestion
- `tt create`: added single instance application template.
```

Comment (0x501D) [reply CHANGELOG.md]: Fixed

**Review by oleg-jukovec — APPROVED** [review]

**Pull request merged**